barbell/src/index.ts

```diff
@@ -22,15 +24,22 @@ bot.defineAction(newUserAction)
 
 const openGarageAction = new Action({
 	name: "Open Garage",
-	handler: async (io) => {
-		await io.output.markdown("Opening garage...")
+	handler: async ({ io, userId, channelId, channelType }) => {
```

Comment:
can we expose these fields in alternative ways? seems verbose for end user.

Reply:
end user doesn't have to use all these fields, which is why i made it a dictionary function input. if you don't need all the inputs, say you just need `io`, you can just write `handler: async ({ io }) => {}`

Reply:
I'd want to have like a single param (or 2 if technically required) that will expose all of these things. I see a situation where this would be onerous for the user and something we can make easy.